### ISSUE TITLE
Anyone having experience with MacBook Pro 2010 13" (Mac-F222BEC8) and Continuity Features (preferably AirDrop)? Always running into boot loop...

### ISSUE THREAD

**Issue by fussel132** [dokterdok/Continuity-Activation-Tool#478]
- OS Version: MacOS Catalina (patched, 10.15.7)
- Mac model: MacBook Pro 2010 13" (MacBookPro7,1 - Mac-F222BEC8)
- Dongle or replaced card?: Yes, USB BT 4.0 Dongle
- Used dongle/card: [This one](https://www.amazon.de/dp/B07NQ5YGDW/)

**Your Issue:**

I tried several steps while trying to activate any Continuity Features under patched MacOS 10.15 (by [dosdude1](https://github.com/dosdude1/macos-catalina-patcher)). The Continuity-Activation-Tool (I also tried the beta) always ends up in boot looping. As far as I am concerned, there are reports of getting AirDrop, Handoff & co. working on my exact model, but I never got any further than the boot loop.

**Steps done to try to fix the issue:**

Of course, I searched for alternatives, modding kexts on my own was also a thing, but that never changed anything. The only change was either a boot loop or the new bluetooth version (4.0) in the system report. The steps [here](https://forums.macrumors.com/threads/macbook-pro-2010-15-with-bcm94331pciebt4cax-handoff-airdrop-continuity-solved.1786829/?post=19841481#post-19841481) were close, didn't work. But [here](https://forums.macrumors.com/threads/macos-10-15-catalina-on-unsupported-macs.2183772/page-282?post=28016436#post-28016436) was a screenshot attached, everything looked like my setup and he even said it would be possible with a BT4.0 dongle, but his steps or tools also doesn't work for me. I tried to contact this guy but macrumors has not enabled my account since a week. I wanted to try first with the dongle, whether the whole thing works at all, because I am not sure yet also because of the size whether a BCM94331PCIEBT4CAX - card fits at all into the MBP. And, of course, it is very expensive, compared to the dongle.
To sum things up, is there any way to at least get AirDrop working? Is it because of the dongle and such a [card](https://www.ebay.de/itm/Apple-MacBook-Pro-Unibody-A1278-A1286-A1297-607-8958-Bluetooth-WiFi-Adapter-4-0/323272624510) is required or will it simply not work with the 13" 2010 MBP?

Thanks for your time, 
Fussel132

*If you need further information etc. just ask, I'll provide them. I really like to use AirDrop with my iPhone, it's so comftable :)

**Comment by fussel132** [comment]
Or is there maybe someone that could mod the neccesary kexts for me to try again? Maybe I am doing that wrong or so... 
Greetings
